operator/pkg/name/name.go

```diff
@@ -86,6 +86,15 @@ const (
 	IstioOperatorCustomResourceName ComponentName = "IstioOperatorCustomResource"
 )
 
+var IstioComponentSuccessIcons = map[ComponentName]string{
+	IstioBaseComponentName: "⛵️",
+	PilotComponentName:     "🧠",
+	CNIComponentName:       "🚦",
```

Comment:
Do you think it makes sense to use the [CNI icon](https://www.cni.dev/)?

Reply:
Ideally that'd be great but I don't _think_ there is a unicode character for it. I'll take another look though.

Reply:
I tried 🪢 which sort of looks like ethernet cables being woven